.goreleaser.yaml

```diff
@@ -173,3 +173,27 @@ dockers:
       - --label=org.opencontainers.image.revision={{ .FullCommit }}
       - --label=org.opencontainers.image.version={{ .Version }}
       - --label=org.opencontainers.image.vendor=Axiom, Inc.
+
+aurs:
+  -
+    name: axiom-bin
+    ids:
+      - nix
```

Comment:
```suggestion
      - linux
```

As `linux` is the ID for the linux build we create (see above in this config file).